virtualfish/virtual.fish

```diff
@@ -442,19 +445,44 @@ function __vf_help --description "Print VirtualFish usage information"
 end
 
 function __vf_globalpackages --description "Toggle global site packages"
-  if set -q VIRTUAL_ENV
-      # use site-packages/.. to avoid ending up in python-wheels
-      pushd $VIRTUAL_ENV/lib/python*/site-packages/..
-      if test -e $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-        echo "Enabling global site packages"
-        command rm $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-      else
-        echo "Disabling global site packages"
-        touch $VIRTUALFISH_GLOBAL_SITE_PACKAGES_FILE
-      end
-      popd
-    else
-        echo "No virtualenv is active."
```

Comment:
Previously, running `vf globalpackages` without an activated virtual environment would show a notification to that effect, but I think that notification was (accidentally?) removed in this PR. Perhaps add it back at the end, with the following message?
```
else
    echo "Cannot toggle global site packages without an active virtual environment."
```

Reply:
Sorry about that, fixed in 82aeb99. I kept the original message for backward compatibility but am not opposed to something more verbose.